python/sglang/srt/server_args.py

```diff
@@ -175,14 +175,14 @@ def __post_init__(self):
             logger.info("Automatically adjust --chunked-prefill-size for small GPUs.")
 
         # Choose kernel backends
-        if not is_flashinfer_available():
-            self.attention_backend = "triton"
-            self.sampling_backend = "pytorch"
-
         if self.attention_backend is None:
-            self.attention_backend = "flashinfer"
+            self.attention_backend = (
+                "flashinfer" if is_flashinfer_available() else "triton"
+            )
         if self.sampling_backend is None:
-            self.sampling_backend = "flashinfer"
+            self.sampling_backend = (
+                "flashinfer" if is_flashinfer_available() else "pytorch"
+            )
 
```

Comment:
add something like
```python
if self.attention_backend == "torch_native":
    logger.info("Cuda graph is disabled because of using torch native attention backend")
    self.disable_cuda_graph = True
```

Reply:
fixed, thanks